_content/booklets/buddhist-perspective-on-time-and-space_hsingyun.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "The Buddhist Perspective on Time and Space"
+authors:
+  - hsingyun
+translator: "Amy Lam and Jeanne Tsai"
+editor: "Fo Guang Shan International Translation Center"
+external_url: "https://www.fgsitc.org/wp-content/uploads/2023/04/A17-The-Buddhist-Perspective-on-Time-and-Space-2019.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1AvK3oc6NJbrcUKSqGWjl8YL4meq0RYCN/view?usp=drivesdk"
+course: cosmology
+tags:
+  - time
+  - space
```

Comment:
The "space" tag is more for scientific cosmology... "place" is the tag for subjective space.  For things tagged with both "time" and "place", however, "wider" is the appropriate umbrella tag 🏖️